lighthouse-core/gather/gatherers/global-listeners.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview
+ * A gatherer to collect information about the event listeners registered on the
+ * global object. For now, the scope is narrowed to events that occur on and
+ * around page unload, but this can be expanded in the future.
+ */
+
+const Gatherer = require('./gatherer.js');
+
+class GlobalListeners extends Gatherer {
```

Comment:
the artifact name doesnt mention Unload but it is scoped to just them.

If you dont wanna include all of them then maybe just change the filter fn to _filterForAllowlistedTypes ?

Reply:
I talked him out of a `UnloadListeners` artifact so that's on me :)

I like the `_filterForAllowlistedTypes` change though 👍